app/src/utils/DatasetMetadata.ts

```diff
@@ -0,0 +1,14 @@
+/* TODO: These are not yet comprehensive, final interfaces */
+
+export interface DatasetMetadata {
+  id: string;
+  name: string;
+  description: string;
+  fields: Array<Field>;
+}
+
+export interface Field {
+  data_type: string;
+  description: string;
+  data_source_id: string;
```

Comment:
I would rename/add things here per the API definition we spec'd in the doc:
- data_source_id => origin_dataset
- description - I imagined this as more like a sentence or paragraph explaining what the column is, and then having a separate "name" field for the actual display name. For example:
```
id: "DP05_0080E",
name: "Population - Asian Only, Non-Hispanic"
description: "The population of people who are Asian and no other race, and not Hispanic"
data_type: "integer",
origin_dataset: "Census - ACS 5 Year",

```

Reply:
I've made those changes- since I'm not using the other fields in the doc (datasets, geographic level, etc) and they're super easy to add later I'm just going to put off adding the rest until we're more certain exactly we want. For example some fields we may want enum values or string etc and those weren't specified in the doc

Reply:
Sounds good!